releases.go

```diff
@@ -365,14 +365,25 @@ func newSchedulerProcess(release *Release, name string, p Process) (*twelvefacto
 		exposure *twelvefactor.Exposure
 		err      error
 	)
+	mergedEnv := twelvefactor.Merge(appEnv, env)
+
+	isExternal := release.App.Exposure == exposePublic
+	if v, ok := mergedEnv["EMPIRE_X_EXPOSURE"]; ok {
```

Comment:
Can we should add some tests similar to https://github.com/remind101/empire/blob/master/tests/empire/empire_test.go#L77-L180 that exercises this?